Rule conflicts between flake8 and black

## Describe the bug

<!-- A clear and concise description of what the bug is. -->

flake8 rule `E203` conflicts black.

## Expected behavior

Rules do not conflict between flake8 and black.

## Steps to Reproduce

Example: https://github.com/github/super-linter/pull/2754

## Logs

https://github.com/github/super-linter/runs/5964437266?check_suite_focus=true (Log of https://github.com/github/super-linter/pull/2754 )

```
2022-04-11 00:28:15 [INFO]   File:[/tmp/lint/.automation/test/python_black/python_good_flake8_1.py]
2022-04-11 00:28:15 [DEBUG]   Found errors. Error code: 1, File type: PYTHON_BLACK, Error on missing exec bit: true
2022-04-11 00:28:15 [ERROR]   Found errors in [black] linter!
2022-04-11 00:28:15 [ERROR]   Error code: 1. Command output:
------
--- /tmp/lint/.automation/test/python_black/python_good_flake8_1.py	2022-04-11 00:21:51.976273 +0000
+++ /tmp/lint/.automation/test/python_black/python_good_flake8_1.py	2022-04-11 00:28:15.484977 +0000
@@ -1,6 +1,6 @@
 """
 test for flake8
 """
 
 m = [1, 2, 3]
-print(m[len("test"):])
+print(m[len("test") :])
would reformat /tmp/lint/.automation/test/python_black/python_good_flake8_1.py

Oh no! 💥 💔 💥
1 file would be reformatted.
------
```

```
2022-04-11 00:28:19 [INFO]   File:[/tmp/lint/.automation/test/python_flake8/python_good_black_1.py]
2022-04-11 00:28:20 [DEBUG]   Found errors. Error code: 1, File type: PYTHON_FLAKE8, Error on missing exec bit: true
2022-04-11 00:28:20 [ERROR]   Found errors in [flake8] linter!
2022-04-11 00:28:20 [ERROR]   Error code: 1. Command output:
------
/tmp/lint/.automation/test/python_flake8/python_good_black_1.py:6:20: E203 whitespace before ':'
------
```

--- a/.automation/test/python_black/python_bad_1.py
+++ b/.automation/test/python_black/python_bad_1.py
@@ -12,7 +12,7 @@ api_url = getenv(API_URL, default='https://api.github.com/graphql' )
 github_token = getenv("GITHUB_TOKEN",
 default=None)
 m = [1, 2, 3]
-print(m[1:2])
+print(m[1:])
 
 if github_token is None
     sys.exit("GitHub Token is not set." +

--- a/.automation/test/python_black/python_bad_1.py
+++ b/.automation/test/python_black/python_bad_1.py
@@ -11,6 +11,8 @@ env = load_dotenv()
 api_url = getenv(API_URL, default='https://api.github.com/graphql' )
 github_token = getenv("GITHUB_TOKEN",
 default=None)
+m = [1, 2, 3]
+print(m[1:2])
 
 if github_token is None
     sys.exit("GitHub Token is not set." +

--- a/.automation/test/python_black/python_bad_1.py
+++ b/.automation/test/python_black/python_bad_1.py
@@ -12,7 +12,7 @@ api_url = getenv(API_URL, default='https://api.github.com/graphql' )
 github_token = getenv("GITHUB_TOKEN",
 default=None)
 m = [1, 2, 3]
-print(m[len('t'):])
+print(m[len("t"):])
 
 if github_token is None
     sys.exit("GitHub Token is not set." +

--- a/.automation/test/python_black/python_bad_1.py
+++ b/.automation/test/python_black/python_bad_1.py
@@ -12,7 +12,7 @@ api_url = getenv(API_URL, default='https://api.github.com/graphql' )
 github_token = getenv("GITHUB_TOKEN",
 default=None)
 m = [1, 2, 3]
-print(m[1:])
+print(m[len('t'):])
 
 if github_token is None
     sys.exit("GitHub Token is not set." +

--- a/.automation/test/python_black/python_good_1.py
+++ b/.automation/test/python_black/python_good_1.py
@@ -11,7 +11,7 @@ env = load_dotenv()
 api_url = getenv("API_URL", default="https://api.github.com/graphql")
 github_token = getenv("GITHUB_TOKEN", default=None)
 m = [1, 2, 3]
-print(m[1 : 2])
+print(m[1 :])
 
 if github_token is None:
     sys.exit(

--- a/.automation/test/python_black/python_good_1.py
+++ b/.automation/test/python_black/python_good_1.py
@@ -11,7 +11,7 @@ env = load_dotenv()
 api_url = getenv("API_URL", default="https://api.github.com/graphql")
 github_token = getenv("GITHUB_TOKEN", default=None)
 m = [1, 2, 3]
-print(m[1 :])
+print(m[len('t') :])
 
 if github_token is None:
     sys.exit(

--- a/.automation/test/python_black/python_good_1.py
+++ b/.automation/test/python_black/python_good_1.py
@@ -11,7 +11,7 @@ env = load_dotenv()
 api_url = getenv("API_URL", default="https://api.github.com/graphql")
 github_token = getenv("GITHUB_TOKEN", default=None)
 m = [1, 2, 3]
-print(m[len('t') :])
+print(m[len("t") :])
 
 if github_token is None:
     sys.exit(

--- a/.automation/test/python_black/python_good_1.py
+++ b/.automation/test/python_black/python_good_1.py
@@ -10,6 +10,8 @@ from dotenv import load_dotenv  # pylint: disable=import-error
 env = load_dotenv()
 api_url = getenv("API_URL", default="https://api.github.com/graphql")
 github_token = getenv("GITHUB_TOKEN", default=None)
+m = [1, 2, 3]
+print(m[1 : 2])
 
 if github_token is None:
     sys.exit(

--- a/.automation/test/python_flake8/python_good_1.py
+++ b/.automation/test/python_flake8/python_good_1.py
@@ -11,7 +11,7 @@ env = load_dotenv()
 api_url = getenv("API_URL", default="https://api.github.com/graphql")
 github_token = getenv("GITHUB_TOKEN", default=None)
 m = [1, 2, 3]
-print(m[1 : 2])
+print(m[1 :])
 
 if github_token is None:
     sys.exit(

--- a/.automation/test/python_flake8/python_good_1.py
+++ b/.automation/test/python_flake8/python_good_1.py
@@ -11,7 +11,7 @@ env = load_dotenv()
 api_url = getenv("API_URL", default="https://api.github.com/graphql")
 github_token = getenv("GITHUB_TOKEN", default=None)
 m = [1, 2, 3]
-print(m[1 :])
+print(m[len('t') :])
 
 if github_token is None:
     sys.exit(

--- a/.automation/test/python_flake8/python_good_1.py
+++ b/.automation/test/python_flake8/python_good_1.py
@@ -11,7 +11,7 @@ env = load_dotenv()
 api_url = getenv("API_URL", default="https://api.github.com/graphql")
 github_token = getenv("GITHUB_TOKEN", default=None)
 m = [1, 2, 3]
-print(m[len('t') :])
+print(m[len("t") :])
 
 if github_token is None:
     sys.exit(

--- a/.automation/test/python_flake8/python_good_1.py
+++ b/.automation/test/python_flake8/python_good_1.py
@@ -10,6 +10,8 @@ from dotenv import load_dotenv  # pylint: disable=import-error
 env = load_dotenv()
 api_url = getenv("API_URL", default="https://api.github.com/graphql")
 github_token = getenv("GITHUB_TOKEN", default=None)
+m = [1, 2, 3]
+print(m[1 : 2])
 
 if github_token is None:
     sys.exit(

--- a/.github/linters/.flake8
+++ b/.github/linters/.flake8
@@ -1,2 +1,3 @@
 [flake8]
+ignore = E203
 max-line-length = 120

--- a/.github/linters/.flake8
+++ b/.github/linters/.flake8
@@ -1,3 +1,3 @@
 [flake8]
-ignore = E203,W503
 max-line-length = 120
+extend-ignore = E203

--- a/.github/linters/.flake8
+++ b/.github/linters/.flake8
@@ -1,3 +1,3 @@
 [flake8]
-ignore = E203
+ignore = E203,W503
 max-line-length = 120

--- a/TEMPLATES/.flake8
+++ b/TEMPLATES/.flake8
@@ -1,2 +1,3 @@
 [flake8]
+ignore = E203
 max-line-length = 120

--- a/TEMPLATES/.flake8
+++ b/TEMPLATES/.flake8
@@ -1,3 +1,3 @@
 [flake8]
-ignore = E203,W503
 max-line-length = 120
+extend-ignore = E203

--- a/TEMPLATES/.flake8
+++ b/TEMPLATES/.flake8
@@ -1,3 +1,3 @@
 [flake8]
-ignore = E203
+ignore = E203,W503
 max-line-length = 120

